### ISSUE TITLE
Add status icons to describe state form

### ISSUE THREAD

**Issue by garethrees** [mysociety/alaveteli#4286]
![screen shot 2017-10-11 at 16 37 41](https://user-images.githubusercontent.com/282788/31450956-b79d07f0-aea2-11e7-98db-69a19f5c298a.png)

This is really hard to parse and figure out what its asking.

Adding the state icons next to each option would help users find the item they're after.

**Comment by HelenWDTK** [comment]
<img width="988" alt="Screenshot 2023-03-25 at 21 11 35" src="https://user-images.githubusercontent.com/120410992/227742217-17be144a-1f3f-43e3-9ec2-ab1a70750111.png">

It'd probably need the alignment tweaking slightly, but this was trivial to do using `<i class="icon-standalone icon_<%= state %>"></i>`. I think it does look better 🙂

**Comment by mdeuk** [comment]
> It'd probably need the alignment tweaking slightly, but this was trivial to do using `<i class="icon-standalone icon_<%= state %>"></i>`. I think it does look better 🙂

It might need a little more space - but, it looks fantastic, and doesn't need any new iconography to be created. Fantastic 👍
